### PR TITLE
feat(BottomSheet): add onBackdropPress handler

### PR DESCRIPTION
**Pull request by kedar09**
**What kind of change does this PR introduce?**

I have added the onBackdropPress function for close bottomsheet.

**Did you add tests for your changes?**
Yes
**If relevant, did you update the documentation?**

**Summary**

I have used [react-native-elements](https://github.com/react-native-elements/react-native-elements) package and I need the onBackdropPress function for bottomsheet like in overlay  onBackdropPress

**Does this PR introduce a breaking change?**
No

**Other information**

### CODE REVIEW

**Comment by khushal87** [comment]
Please resolve merge conflicts.

**Review by khushal87 — COMMENTED** [review]
Please resolve merge conflicts.

File: src/bottomSheet/BottomSheet.tsx [hunk 1]
```diff
@@ -15,23 +16,38 @@ import { RneFunctionComponent } from '../helpers';
...
+      <TouchableWithoutFeedback
+        onPress={onBackdropPress}
+        testID="RNE__Overlay__backdrop"
+      >
+        <View
+          testID="backdrop"
+          style={StyleSheet.flatten([styles.backdrop, backdropStyle])}
+        />
+      </TouchableWithoutFeedback>
+
```
Comment (arpitBhalla) [inline src/bottomSheet/BottomSheet.tsx:50]: Could you use `Pressable`?

**Review by arpitBhalla — APPROVED** [review]

**Comment by arpitBhalla** [comment]
@kedar09 Can you resolve conflict

**Review by arpitBhalla — APPROVED** [review]

**Comment by kedar09** [comment]
> @kedar09 Can you resolve conflict
I have updated the code with the latest code in the next branch and push that changes

**Comment by jorgegvallejo** [comment]
Hope this gets added to the next release :pray:

**Comment by flyingcircle** [comment]
I tried this out in the demo expo app. Did not work.

**Review by flyingcircle — CHANGES_REQUESTED** [review]
Tested this in the demo app. Did not work.

**Comment by kedar09** [comment]
> pointerEvents={'none'}

@arpitBhalla i have used 
style={[StyleSheet.absoluteFill, styles.backdrop, backdropStyle]}
and pointerEvents="box-none"
its worked
```

return (<Modal onRequestClose={onBackdropPress} animationType="slide" transparent={true} visible={isVisible} {...modalProps}>
    <Pressable onPress={onBackdropPress}   
    style={[StyleSheet.absoluteFill, styles.backdrop, backdropStyle]}
     testID="RNE__Overlay__backdrop" />
    
      <SafeAreaView style={StyleSheet.flatten([
        styles.safeAreaView,
        containerStyle && containerStyle,
    ])} {...props} pointerEvents="box-none">
        <View>
          <ScrollView {...scrollViewProps}>{children}</ScrollView>
        </View>
      </SafeAreaView>
    </Modal>);
```

**Comment by arpitBhalla** [comment]
> @arpitBhalla i have used
style={[StyleSheet.absoluteFill, styles.backdrop, backdropStyle]}
and pointerEvents="box-none"
its worked

Pls Update the PR, so maintainers can review.

**Review by ridie001 — APPROVED** [review]

**Review by arpitBhalla — APPROVED** [review]

**Comment by arpitBhalla** [comment]
Thank you for your contribution 🎉

**Pull request merged**